tensorrt_llm/_torch/pyexecutor/model_engine.py

```diff
@@ -414,13 +414,27 @@ def get_cuda_graph_warmup_request(batch_size):
 
         def get_torch_compile_warmup_request(batch_size, num_tokens):
             available_blocks = kv_cache_manager.get_num_free_blocks()
+            max_num_draft_tokens = 0
```

Comment:
The context reqeusts don't need draft tokens, but the generation requests do.
```suggestion
            max_num_draft_tokens = 0
            if self.spec_config is not None and num_tokens==1:
                max_num_draft_tokens = self.spec_config.max_draft_tokens 
```

Reply:
Set max_num_draft_tokens to generation requests.